packages/target-electron/src/ipc.ts

```diff
@@ -374,6 +382,8 @@ export async function copyFileToInternalTmpDir(
 ): Promise<string> {
   const sourceFileName = basename(sourcePath)
   const sourceDir = dirname(sourcePath)
+  // make sure fileName includes only a file name, no path or whatever
+  fileName = basename(normalize(fileName))
```

Comment:
Why is `normalize` needed? Shouldn't `basename()` be enough?

Reply:
was not sure if basename can be "tricked" if it gets an unexpected argument, while normalize is obviously created for "normalizing" but may be I exxagerate...

Reply:
All good. Better safe than sorry!